src/components/user/event.tsx

```diff
@@ -256,14 +256,33 @@ export function Event({
     alias,
     title,
     __typename,
+    id,
   }: EventObject | EventParent) {
     return (
-      <Link href={alias}>{renderTitle(title, __typename as UuidType)}</Link>
+      <Link href={alias}>
+        <>
+          {renderTitle(title, __typename as UuidType, id)}
+          {shouldRenderParent(__typename as UuidType) ? (
+            <>{renderParent(title, __typename as UuidType)}</>
+          ) : (
+            <></>
```

Comment:
you can also just return `null` here. sorry, did not spot that earlier.

Reply:
Good to know! :)